packages/react-native-test-app-msal/android/build.gradle

```
@@ -74,6 +74,11 @@ repositories {
 }
 
 def rnxAuth = findNodeModulesPath("@rnx-kit/react-native-auth")
+def rnxDisableAuthProperty = "com.microsoft.reacttestapp.msal.disableRnxAuthModule"
+def skipAuthModuleRegistration =
+    rnxAuth != null &&
```

Actually, on closer inspection, this looks inverted. Shouldn't this be something like:
```suggestion
    rnxAuth == null ||
```